stories/Breadcrumbs.stories.mdx

```diff
@@ -109,3 +119,10 @@ export const CurrentPageAsLinkTemplate = () => html`
       {CurrentPageAsLinkTemplate.bind({})}
     </Story>
 </Canvas>
+
+<Canvas>
+    <Story name="Back to Home" args={{
```

Comment:
I would go with "Back to Parent" here.  It may not be a return to the Home in some situations.  One of the uses that I can see for this particular variant will be in list/detail scenarios where someone drills down and needs to return to the list.  We had a few of those cases in AMT where we ended up just using a button in the top left corner to return to the parent page.